src/App.tsx

```diff
@@ -7,6 +7,7 @@ import { MoviesShowsProvider } from './context/Context';
 import Error from './pages/Error/Error';
 import Home from './pages/Home/Home';
 import MovieOrShow from './pages/MovieOrShow/MovieOrShow';
+import Registration from './pages/Registration/Registration';
```

Comment:
Introduce index file. This import will become: 

```suggestion
import Registration from './pages/Registration';
```